packages/cli/src/main/kotlin/elide/tool/cli/cmd/repl/ToolShellCommand.kt

```diff
@@ -487,8 +441,8 @@ import org.graalvm.polyglot.Engine as VMEngine
   // Active line reader.
   private val lineReader: AtomicReference<LineReader?> = AtomicReference(null)
 
-  // All VFS configurators.
-  @Inject lateinit var vfsConfigurators: List<GuestVFS.VFSConfigurator>
```

Comment:
The role of VFS Configurators (mounting platform-specific VFS bundles) has been assumed by the embedded resources API used by the runtime language plugins. These bundles are automatically mounted by the language plugins.